recipes/odl/meta.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+    name: odl
+    version: "0.8.0"
+
+source:
+    url: https://github.com/odlgroup/odl/archive/refs/tags/v0.8.0.tar.gz
+    sha256: dec092ce1f152707d821146046d4d1441672ad6c22d28dd46e1f77d79c210a38
+
+build:
+    number: 0
+    script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+    build:
+        - python
+        - setuptools >=65.6
+
+    host:
+        - python
+        - setuptools
+        - nomkl  # [not win]
+        - future >=0.16
+        - numpy >=1.19, <1.27
+        - scipy >=1.1
+        - packaging >=17.0
+
+    run:
+        - python
+        - future >=0.16
+        - numpy >=1.19
+        - scipy >=1.1
+        - packaging >=17.0
+        - matplotlib-base >=3.4
+
+test:
+    requires:
+        - nomkl  # [not win]
```

Comment:
```suggestion
```

Reply:
Done in [6f585a9](https://github.com/conda-forge/staged-recipes/pull/27504/commits/6f585a97e5d82eb7001080118f9f8991647a3033)